STSMACOM-828 - Extend ViewMetadata component to accept a render function as a prop (#1467)
folio-org/stripes-smart-components: CHANGELOG.md

* Use `react-quill` compatible with `react` `v18`. Refs STSMACOM-821.
* `<ControlledVocab>` - pass override headers to PUT method, new prop to hide "New" button. Refs STSMACOM-825.
* `<SearchAndSort>` - re-position Advanced search button when search panel does not have enough space. Refs STSMACOM-830.

> * Extend `ViewMetadata` component to accept a render function as a prop. Refs STSMACOM-828.

Reviewer: ```suggestion
* Extend `ViewMetadata` component to accept a child render function for custom rendering. Refs STSMACOM-828.
```
